helm-chart/renku-graph/charts/jena/templates/configmap.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jena-shiro-ini
+  labels:
+    app: {{ template "jena.name" . }}
+    chart: {{ template "jena.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  shiro.ini: |-
+    [main]
+    # Development
+    ssl.enabled = false
+
+    plainMatcher=org.apache.shiro.authc.credential.SimpleCredentialsMatcher
+    iniRealm.credentialsMatcher = $plainMatcher
+
+    [users]
+    admin={{ .Values.admin.password }}
+    renkuuser={{ .Values.renkuuser.password }}
```

Comment:
or just `renku`?

Reply:
Yep, seems to be unnecessarily overcomplicated now. I'll change it.